doc/introduction/circuits.rst

```diff
@@ -98,8 +98,7 @@ simulators; additional devices can be installed as plugins (see
 choice of a device significantly determines the speed of your computation, as well as
 the available options that can be passed to the device loader.
 
-Device options
-~~~~~~~~~~~~~~
+**Device options**
```

Comment:
How come that here the formatting changed?

Reply:
![image](https://user-images.githubusercontent.com/24476053/73514079-480ed400-43be-11ea-95cf-ade64d23191d.png)

Reply:
Looks like this now

Reply:
Ah, because it was showing in the table of contents on the right, and looked a bit weird: 
![image](https://user-images.githubusercontent.com/2959003/73523385-b2496800-4402-11ea-9764-3465b452325e.png)

Reply:
Sounds reasonable, nice!